openapi/paths/risk-scores.yaml

```
@@ -0,0 +1,62 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Risk Score
+  summary: Retrieve risk score rules
+  operationId: GetRiskScoreRules
+  x-sdk-operation-name: getAll
+  security:
+    - SecretApiKey: []
+    - JWT: []
+    - ApplicationJWT: []
+  description: Retrieve risk score rule.
+  responses:
+    '200':
+      description: Risk score rules retrieved.
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/RiskScore/RiskScoreRules.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+put:
+  x-products:
+    - Core
+  tags:
+    - Risk Score
+  summary: Change Risk Score rules
```

nitpick: I think we should use 'modify' or 'edit' instead of 'change' for these summaries  

I think that 'change' may be ambiguous 

```suggestion
  summary: Modify risk score rules
```